File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/PlaylistEditEndpoint.java
package com.github.kotvertolet.youtubejextractor.models.test;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class PlaylistEditEndpoint{

	@SerializedName("playlistId")
	private String playlistId;

	@SerializedName("actions")
	private List<ActionsItem> actions;

	public void setPlaylistId(String playlistId){
		this.playlistId = playlistId;
	}

	public String getPlaylistId(){
		return playlistId;
	}

	public void setActions(List<ActionsItem> actions){
		this.actions = actions;
	}

	public List<ActionsItem> getActions(){
		return actions;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/PlaylistEditEndpoint.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class PlaylistEditEndpoint : Parcelable, Serializable {
    @SerializedName("playlistId")
    var playlistId: String? = null

    @SerializedName("actions")
    var actions: List<ActionsItem>? = null
    override fun toString(): String {
        return "PlaylistEditEndpoint{" +
                "playlistId = '" + playlistId + '\'' +
                ",actions = '" + actions + '\'' +
                "}"
    }
}